backend/models/core/Entity.js

```diff
@@ -86,79 +87,84 @@ class Entity {
                       entity_types.slug = :entity_type_slug  
                   ORDER BY entities.id, attributes.\`order\` ASC
                   `;
-                
-      const data = await db.executeStatement(sql, [
-          {name: 'entity_type_slug', value:{stringValue: entity_type_slug}},
-      ]);
-      
-      return data.records.map(([
-          {longValue: id},
-          {longValue: entity_type_id},
-          {stringValue: entity_type_name},
-          {stringValue: entity_type_slug},
-          {stringValue: entities_slug},
-          {stringValue: attributes_name},
-          {stringValue: attributes_type},
-          {longValue: attributes_order},
-          {stringValue: value_string},
-          {stringValue: value_long_string},
-          {longValue: value_integer},
-          {stringValue: value_datetime},
-          {stringValue: value_double}
-        ]) => ({
-          id, 
-          entity_type_id,
-          entity_type_name, 
-          entity_type_slug, 
-          entities_slug, 
-          attributes_name, 
-          attributes_type, 
-          attributes_order, 
-          value_string,
-          value_long_string,
-          value_integer,
-          value_datetime,
-          value_double
-        })); 
+
+    const data = await db.executeStatement(sql, [
+      { name: "entity_type_slug", value: { stringValue: entity_type_slug } },
+    ]);
+
+    return data.records.map(
+      ([
+        { longValue: id },
+        { longValue: entity_type_id },
+        { stringValue: entity_type_name },
+        { stringValue: entity_type_slug },
+        { stringValue: entities_slug },
+        { stringValue: attributes_name },
+        { stringValue: attributes_type },
+        { longValue: attributes_order },
+        { stringValue: value_string },
+        { stringValue: value_long_string },
+        { longValue: value_integer },
+        { stringValue: value_datetime },
+        { stringValue: value_double },
+      ]) => ({
+        id,
+        entity_type_id,
+        entity_type_name,
+        entity_type_slug,
+        entities_slug,
+        attributes_name,
+        attributes_type,
+        attributes_order,
+        value_string,
+        value_long_string,
+        value_integer,
+        value_datetime,
+        value_double,
+      })
+    );
   }
 
-  static async findByPageAndEntry({entity_type_slug , entry, page }) {
+  static async findByPageAndEntry({ entity_type_slug, entry, page }, queries) {
```

Comment:
`where` and `findByPageAndEntry` is essentially the same function now. Let's just combine them both as `where({ entity_type_slug, entry, page, filters})`